test/enhance/test_adjust.py

```diff
@@ -466,12 +466,12 @@ def test_gradcheck(self, device, dtype):
 
 class TestEqualize:
     def test_shape_equalize(self, device, dtype):
-        bs, channels, height, width = 1, 3, 4, 5
+        channels, height, width = 3, 4, 5
```

Comment:
can you add `@pytest.mark.parametrize` and test different shapes ?

Reply:
@edgarriba sure thing.